frontend/src/components/Newsletter/Newsletter.tsx

```diff
@@ -1,22 +1,28 @@
+import Alert from '@mui/material/Alert'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Container from '@mui/material/Container'
+import TextField from '@mui/material/TextField'
+import Typography from '@mui/material/Typography'
+import { AxiosResponse } from 'axios'
 import React, { useEffect, useRef, useState } from 'react'
-import Button from '../Button/Button'
-import Input from '../Input/Input'
 import { Link, useLocation } from 'react-router-dom'
-import './Newsletter.css'
```

Comment:
Since we are not using the ./Newsletter.css file should we delete it or keep it in case we'll transition back to plain js? This also applies to other files that do not use their old css file